honeybread-core/src/main/java/com/whatsub/honeybread/core/domain/orderpricedeliverytip/OrderPriceDeliveryTipRepositoryImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.whatsub.honeybread.core.domain.orderpricedeliverytip;
+
+import com.querydsl.core.types.Predicate;
+import com.whatsub.honeybread.core.domain.model.Money;
+import com.whatsub.honeybread.core.support.QuerydslRepositorySupport;
+
+import java.util.Optional;
+
+import static com.whatsub.honeybread.core.domain.orderpricedeliverytip.QOrderPriceDeliveryTip.orderPriceDeliveryTip;
+
+public class OrderPriceDeliveryTipRepositoryImpl extends QuerydslRepositorySupport implements OrderPriceDeliveryTipRepositoryCustom {
+
+    public OrderPriceDeliveryTipRepositoryImpl() {
+        super(OrderPriceDeliveryTip.class);
+    }
+
+    @Override
+    public Optional<OrderPriceDeliveryTip> getTipByOrderPrice(final long storeId, final Money target) {
+        return Optional.ofNullable(
+            from(orderPriceDeliveryTip)
+                .where(
+                    eqStoreId(storeId),
+                    includePrice(target))
+                .fetchOne());
+    }
```

Comment:
어디까지가 어느 메소드 호출부 인지 구분하기 힘들수 있을것 같은데 코드 정리 한번 필요할 것 같습니다
```suggestion
    public Optional<OrderPriceDeliveryTip> getTipByOrderPrice(final long storeId, final Money target) {
        return Optional.ofNullable(
            from(orderPriceDeliveryTip)
                .where(
                    eqStoreId(storeId),
                    includePrice(target)
                ).fetchOne()
       );
    }
```

Reply:
이것도 확인해주셔서 감사합니다~! 반영했습니다~!